### PR TITLE
[APAF-1316] enhance documentation + demos

### PR DESCRIPTION
**Pull request by josie-ku-awx**

### CODE REVIEW

File: integrations/react/src/components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
+/**
```
Comment (josie-ku-awx) [inline integrations/react/src/components/Card.jsx:1]: Please review this file ~

File: README.md [hunk 2]
```diff
@@ -6,6 +6,17 @@ Our goal is to make it easy for you (Merchants) to use our platform, as easy as
 
 Before getting start, check out what the Airwallex Checkout elements look like here: [https://demo-pacheckoutdemo.airwallex.com/](https://demo-pacheckoutdemo.airwallex.com/).
 
+## Creating a Payment Intent
+
+Throughout this demo, you will be asked to replace your intent secrets to properly enact a payment checkout:
```
Comment (CharlieLangCL) [inline README.md:11]:
```suggestion
Throughout this demo, you will be asked to replace your intent secrets to properly enact a payment checkout, more details can be found [Create a PaymentIntent](https://www.airwallex.com/docs/api#/Payment_Acceptance/Payment_Intents/_api_v1_pa_payment_intents_create/post):
```

**Review by CharlieLangCL — APPROVED** [review]

File: integrations/next-ts/pages/dropin/index.tsx [hunk 3]
```diff
@@ -2,47 +2,71 @@ import { useEffect } from 'react';
...
 } from 'airwallex-payment-elements';
 
-const intent_id = 'replace-with-your-intent-id';
-const client_secret = 'replace-with-your-client-secret';
+// const intent_id = 'replace-with-your-intent-id';
+// const client_secret = 'replace-with-your-client-secret';
+const intent_id = 'int_d39jDcBwzndyDsBI5zPwvxLXeXS';
+const client_secret =
+  'eyJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTEwNDM4MTQsImV4cCI6MTYxMTA0NzQxNCwiYWNjb3VudF9pZCI6ImZmMjI1YzEzLWQ5ODEtNDU2Yy1iZjk3LWYzODIxYzg1YTEyMiIsImRhdGFfY2VudGVyX3JlZ2lvbiI6IkhLIiwiaW50ZW50X2lkIjoiaW50X2QzOWpEY0J3em5keURzQkk1elB3dnhMWGVYUyJ9.BQ7A0rr2Z22thv_vhiEYuUGFUn3HgT4H9d14RgEpViw';
+
```
Comment (CharlieLangCL) [inline integrations/next-ts/pages/dropin/index.tsx:13]: should replace those with place holders

File: integrations/next-ts/pages/full-featured-card/index.tsx [hunk 4]
```diff
@@ -5,44 +5,66 @@ import {
   ElementType,
 } from 'airwallex-payment-elements';
 
-const intent_id = 'replace-with-your-intent-id';
-const client_secret = 'replace-with-your-client-secret';
+// const intent_id = 'replace-with-your-intent-id';
+// const client_secret = 'replace-with-your-client-secret';
+const intent_id = 'int_jHQ3oq7PztYfdpeVtzMMlxg8Vbf';
+const client_secret =
+  'eyJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTEwNDc5NTYsImV4cCI6MTYxMTA1MTU1NiwiYWNjb3VudF9pZCI6ImZmMjI1YzEzLWQ5ODEtNDU2Yy1iZjk3LWYzODIxYzg1YTEyMiIsImRhdGFfY2VudGVyX3JlZ2lvbiI6IkhLIiwiaW50ZW50X2lkIjoiaW50X2pIUTNvcTdQenRZZmRwZVZ0ek1NbHhnOFZiZiJ9.aredlqspxqzCBHePKIQOZWgDLS6rp5DIYYGPR4KZtMk';
```
Comment (CharlieLangCL) [inline integrations/next-ts/pages/full-featured-card/index.tsx:12]: Same here

File: integrations/next-ts/pages/hpp/index.tsx [hunk 5]
```diff
@@ -1,29 +1,51 @@
 import { redirectToCheckout, loadAirwallex } from 'airwallex-payment-elements';
 
-const intent_id = 'replace-with-your-intent-id';
-const client_secret = 'replace-with-your-client-secret';
-
-async function redirectHpp(){
+const intent_id = 'int_d39jDcBwzndyDsBI5zPwvxLXeXS';
+const client_secret =
+  'eyJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTEwNDM4MTQsImV4cCI6MTYxMTA0NzQxNCwiYWNjb3VudF9pZCI6ImZmMjI1YzEzLWQ5ODEtNDU2Yy1iZjk3LWYzODIxYzg1YTEyMiIsImRhdGFfY2VudGVyX3JlZ2lvbiI6IkhLIiwiaW50ZW50X2lkIjoiaW50X2QzOWpEY0J3em5keURzQkk1elB3dnhMWGVYUyJ9.BQ7A0rr2Z22thv_vhiEYuUGFUn3HgT4H9d14RgEpViw';
```
Comment (CharlieLangCL) [inline integrations/next-ts/pages/hpp/index.tsx:5]: also here

File: integrations/next-ts/pages/split-card/index.tsx [hunk 6]
```diff
@@ -1,35 +1,70 @@
...
+  EventDetail,
 } from 'airwallex-payment-elements';
 
-const intent_id = 'replace-with-your-intent-id';
-const client_secret = 'replace-with-your-client-secret';
+// const intent_id = 'replace-with-your-intent-id';
+// const client_secret = 'replace-with-your-client-secret';
+const intent_id = 'int_d39jDcBwzndyDsBI5zPwvxLXeXS';
+const client_secret =
+  'eyJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTEwNDM4MTQsImV4cCI6MTYxMTA0NzQxNCwiYWNjb3VudF9pZCI6ImZmMjI1YzEzLWQ5ODEtNDU2Yy1iZjk3LWYzODIxYzg1YTEyMiIsImRhdGFfY2VudGVyX3JlZ2lvbiI6IkhLIiwiaW50ZW50X2lkIjoiaW50X2QzOWpEY0J3em5keURzQkk1elB3dnhMWGVYUyJ9.BQ7A0rr2Z22thv_vhiEYuUGFUn3HgT4H9d14RgEpViw';
```
Comment (CharlieLangCL) [inline integrations/next-ts/pages/split-card/index.tsx:14]: also here :)

File: integrations/next-ts/pages/wechat/index.tsx [hunk 7]
```diff
@@ -2,18 +2,25 @@ import { useEffect } from 'react';
...
+  ElementType,
 } from 'airwallex-payment-elements';
 
-const intent_id = 'replace-with-your-intent-id';
-const client_secret = 'replace-with-your-client-secret';
+// const intent_id = 'replace-with-your-intent-id';
+// const client_secret = 'replace-with-your-client-secret';
+const intent_id = 'int_6QSnVZ2QzlZ3SGghOzHN6OoFdyY';
+const client_secret =
+  'eyJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTEwNDgzMTIsImV4cCI6MTYxMTA1MTkxMiwiYWNjb3VudF9pZCI6ImZmMjI1YzEzLWQ5ODEtNDU2Yy1iZjk3LWYzODIxYzg1YTEyMiIsImRhdGFfY2VudGVyX3JlZ2lvbiI6IkhLIiwiaW50ZW50X2lkIjoiaW50XzZRU25WWjJRemxaM1NHZ2hPekhONk9vRmR5WSJ9.bMy3hrS-m8dXiY79CaQD5xvcqA7IwajpkFKUSlmZKOA';
```
Comment (CharlieLangCL) [inline integrations/next-ts/pages/wechat/index.tsx:12]: also here

File: integrations/node/.env_copy [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# An Example Of Environment Variables.
+
+# Airwallex API keys.
+# Use your test keys for development and live keys for real charges in production.
+
+AIRWALLEX_API_KEY=1105eacc146c413c43648e0f15764eb873810185ccea0d18e1a8b14ab97bb4e8d6cf219672c775e25f89d5267b4d1793
+AIRWALLEX_CLIENT_ID=MSKOSA7KTo2JdEnGoQmcCQ
```
Comment (CharlieLangCL) [inline integrations/node/.env_copy:7]: lets using place holder here too

File: integrations/vue/src/components/wechat.vue [hunk 9]
```diff
@@ -1,31 +1,38 @@
...
 <script>
 import { createElement, loadAirwallex } from 'airwallex-payment-elements';
 
-const intentid = 'replace-with-your-intent-id';
-const client_secret = 'replace-with-your-client-secret';
+// const intent_id = 'replace-with-your-intent-id';
+// const client_secret = 'replace-with-your-client-secret';
+const intent_id = 'int_6FCFmYc6z5XALntrCzqhgF1yhFU';
+const client_secret =
+  'eyJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTA5NjMzNDIsImV4cCI6MTYxMDk2Njk0MiwiYWNjb3VudF9pZCI6ImZmMjI1YzEzLWQ5ODEtNDU2Yy1iZjk3LWYzODIxYzg1YTEyMiIsImRhdGFfY2VudGVyX3JlZ2lvbiI6IkhLIiwiaW50ZW50X2lkIjoiaW50XzZGQ0ZtWWM2ejVYQUxudHJDenFoZ0YxeWhGVSJ9.JP6aZyoipwEjAsMxwYb8-0N-pmTmA1WovABOrJ5FzVA';
```
Comment (CharlieLangCL) [inline integrations/vue/src/components/wechat.vue:16]: same here

File: integrations/node/README.md [hunk 10]
```diff
@@ -1,29 +1,44 @@
...
+
+- **Create a payment intent** at [http://localhost:3002/api/v1/intent/create](http://localhost:3002/api/v1/intent/create)
+- **Retrieve a payment intent** at [http://localhost:3002/api/v1/intent/:id](http://localhost:3002/api/v1/intent/:id)
+  - remember to change id to a live intentId in the url
+
+Find more details about the Airwallex API [here](https://www.airwallex.com/docs/api#/Payment_Acceptance/Payment_Intents/_api_v1_pa_payment_intents_create/post)!
+
+## Author
+
+Shirley Chen
```
Comment (shirly-chen-awx) [inline integrations/node/README.md:44]: shirly  lol～

**Review by CharlieLangCL — APPROVED** [review]

File: integrations/react-ts/src/App.css [hunk 11]
```diff
@@ -0,0 +1,86 @@
+.App {
```
Comment (shirly-chen-awx) [inline integrations/react-ts/src/App.css:1]: do we need a explanation here.  just some common css styles not necessary when doing integration

File: integrations/react-ts/src/components/dropIn.tsx [hunk 12]
```diff
@@ -1,44 +1,116 @@
...
+      });
+      // STEP 5: Mount the drop-in element to the empty container created previously
+      element?.mount('dropIn'); // This 'dropIn' id MUST MATCH the id on your empty container created in Step 3
     });
-    const onSuccess = (event: CustomEvent) => {
-      /*
-        ... Handle event
-      */
+
+    // STEP #6: Add an event listener to handle events when the element is mounted
```
Comment (shirly-chen-awx) [inline integrations/react-ts/src/components/dropIn.tsx:56]: why some step has `#` some not?

File: integrations/react-ts/src/components/fullFeatureCard.tsx [hunk 13]
```diff
@@ -1,44 +1,116 @@
...
     window.addEventListener('onSuccess', onSuccess as EventListener);
+    window.addEventListener('onError', onError as EventListener);
     return () => {
+      window.removeEventListener('onReady', onReady as EventListener);
       window.removeEventListener('onSuccess', onSuccess as EventListener);
+      window.removeEventListener('onError', onError as EventListener);
     };
   }, []);
 
+  const containerStyle = {
```
Comment (shirly-chen-awx) [inline integrations/react-ts/src/components/fullFeatureCard.tsx:91]: May need `Example:`

File: integrations/node/README.md (hunk 10 above)
Comment (josie-ku-awx) [reply integrations/node/README.md:44]: sorry ~ will fix

File: integrations/react-ts/src/components/dropIn.tsx (hunk 12 above)
Comment (josie-ku-awx) [reply integrations/react-ts/src/components/dropIn.tsx:56]: inconsistencies, will fix

**Pull request merged**